README.md

```diff
@@ -799,6 +799,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
 
   - [Yahya Khan](https://github.com/yahyakhn)
   - [Yamini Singh](https://github.com/yamini-singh169)
+  - [Yamini Kaushal](https://github.com/Yamini09-code)
```

Comment:
Please add your name in line 801, after Yahya Khan.

Reply:
Sure